app/assets/javascripts/shopify_app/storage_access.js

```diff
@@ -0,0 +1,116 @@
+(function() {
+  var ACCESS_GRANTED_STATUS = 'access granted';
```

Comment:
It looks like it's being passed to `redirectToAppTLD` which is using it to construct a URL and send a postMessage which I guess redirects via the EASDK. Why not just use window.parent? It seems like it would simplify this a lot.

Reply:
I think the idea is that since the windows are on two different domains, we can't use `window.parent.location`, we have to send a message to the top frame asking it to redirect.